mmdet/apis/ote/sample/sample.py

```diff
@@ -141,6 +145,38 @@ def main(args):
         performance = openvino_task.evaluate(resultset)
         logger.info(str(performance))
 
+        logger.info('Run POT optimization')
+        optimized_model = OptimizedModel(
+            NullProject(),
+            NullModelStorage(),
+            dataset,
+            environment.get_model_configuration(),
+            ModelOptimizationType.POT,
+            optimization_methods=[],
+            optimization_level={},
+            precision=[ModelPrecision.FP16],
```

Comment:
I thought, that produced model should be INT8, not FP16. What am I missing?

Reply:
Yep. Fixed